### PR TITLE
Strip "@" from "r=@username_foobar"

### PR DESCRIPTION
**Pull request by Centril**
Fixes https://github.com/rust-lang/homu/issues/60

N.B. I have not written python in a long long time... (but I tested the function locally in a REPL). Please double check this code. =)

### CODE REVIEW

File: homu/parse_issue_comment.py [hunk 1]
```diff
@@ -16,7 +20,7 @@ def __init__(self, action):
     def approve(cls, approver, commit):
         command = cls('approve')
         command.commit = commit
-        command.actor = approver
+        command.actor = strip_at_prefix(approver)
```
Comment (kennytm) [inline homu/parse_issue_comment.py]: Perhaps just use

```suggestion
        command.actor = approver.lstrip('@')
```

Comment (Centril) [reply homu/parse_issue_comment.py]: Done =)

File: homu/parse_issue_comment.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def __init__(self, action):
     def approve(cls, approver, commit):
         command = cls('approve')
         command.commit = commit
-        command.actor = approver
+        command.actor = approver.lstrip('@')
```
Comment (RalfJung) [inline homu/parse_issue_comment.py]: So would this also strip `@@RalfJung`? I guess GH names can't start with an `@` so it does not matter.

Comment (Centril) [reply homu/parse_issue_comment.py]: Yeah apparently that's what `.lstrip(string)` does -- I agree it does not matter much.

**Pull request merged**